pkg/client/errors/errors.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2021 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"github.com/codenotary/immudb/pkg/api/schema"
+	"google.golang.org/grpc/status"
+)
+
+type ImmuError interface {
```

Comment:
Can we add some comment to this interface? That should make things clear for users of immudb SDK.
The PR description is pretty nice, I'd copy some parts of it over here.

Reply:
Sure, I'm waiting your feedbacks then I will add comments when we agree on the implementation